tools/cluster.py

```diff
@@ -202,7 +201,6 @@ def launch(cluster_name, slaves,
            key_id=default_key_id, region=default_region,
            zone=default_zone, instance_type=default_instance_type,
            ondemand=False, spot_price=default_spot_price,
-           user_data=default_user_data,
```

Comment:
Why remove this feature? Are you sure no one in the world is using it? You can make the default user data a no-op, but can't remove the custom user data feature.

By the way, we are one of the users of the custom user data, see job_runner.py